packages/data-point/lib/reducer/factory.js

```diff
@@ -6,21 +6,41 @@ const ReducerPath = require('../reducer-path')
 const ReducerFunction = require('../reducer-function')
 const ReducerObject = require('../reducer-object')
 const ReducerEntity = require('../reducer-entity')
+const ReducerList = require('../reducer-list')
 const util = require('util')
 
 const reducerTypes = [ReducerPath, ReducerFunction, ReducerEntity]
 
+/**
+ * this is here because ReducerLists can be arrays or | separated strings
+ * @param {*} source
+ * @returns {Array<reducer>|reducer}
+ */
+function dealWithPipeOperators (source) {
+  let result = ReducerList.parse(source)
+  if (result.length === 1) {
+    // do not create a ReducerList that only contains a single reducer
+    result = result[0]
+  }
+
+  return result
+}
+
 /**
  * parse reducer
- * @param {Function} createTransform
  * @param {*} source
  * @throws if source is not a valid type for creating a reducer
  * @return {reducer}
  */
-function create (createTransform, source) {
-  // ReducerObject requires an extra parameter, so it's not included in the reducerTypes array
+function createReducer (source) {
```

Comment:
removing the `createTransform` parameter is a breaking change because the function is being exported here https://github.com/raingerber/data-point/blob/remove-transform-expressions/packages/data-point/lib/helpers/helpers.js#L44